documentation/modules/con-user-model.adoc

```diff
@@ -31,8 +31,8 @@ The `spec.authentication` object defines how the user is authenticated, whereas
 
 The supported values for the authentication type are `password` and `federated`. 
 
-NOTE: Currently `password` authentication type users can access both applications and the {ProductName} Console, whereas
-`federated` authentication type users can only access the {ProductName} Console.
+NOTE: Currently `password` authentication type users can only access messaging endpoints, whereas
+`federated` authentication type users can only access the {ProductName} Console. 
 
```

Comment:
Would it be better to qualify this in some way, such as saying "In the default configuration...", rather than "Currently...".  Also, is this true when run on OpenShift

Reply:
Yes, this is true in the default configuration on OPenShift. You can work around it by setting the kc-hint parameter when creating the address space, but I kind of feel it adds some unnecessary complications to mention it.